docker/images/integration-tests/Dockerfile

```diff
@@ -1,19 +1,14 @@
-FROM quay.io/mozmar/base
```

Comment:
The request in the infra issue was to use `FROM python:2.7-slim`. The ones in the mozmeao repo are just examples. We're moving away from all of our base images as we're not doing well enough keeping up with security updates as the official images are.

Reply:
Ah, my misunderstanding, thanks @pmac, I'll update this PR accordingly.

Reply:
I've added a clarifying comment with an example in the [original issue](https://github.com/mozmeao/infra/issues/641) if that helps. Thanks for jumping on this!